components/tools/OmeroPy/test/unit/clitest/test_import.py

```diff
@@ -235,9 +241,127 @@ def new_client(x):
         ice_config.write('omero.host=%s\nomero.port=%g' % (
             hostname, (port or 4064)))
         monkeypatch.setenv("ICE_CONFIG", ice_config)
-        control.set_login_arguments(self.cli.parser.parse_args(self.args))
+        args = self.cli.parser.parse_args(self.args)
+        command_args = CommandArguments(self.cli, args)
 
         expected_args = ['-s', '%s' % hostname]
         expected_args += ['-p', '%s' % (port or 4064)]
         expected_args += ['-k', '%s' % sessionid]
-        assert control.command_args == expected_args
+        expected_args += ['test.fake']
+        assert command_args.java_args() == expected_args
+
+    def testLogPrefix(self, tmpdir, capfd):
+        fakefile = tmpdir.join("test.fake")
+        fakefile.write('')
+        prefix = tmpdir.join("log")
+
+        self.add_client_dir()
+        self.args += ["-f", "---logprefix=%s" % prefix,
+                      "---file=out", "---errs=errs"]
+        self.args += [str(fakefile)]
+        self.cli.invoke(self.args, strict=True)
+
+        o, e = capfd.readouterr()
+        assert o == ""
+        assert e == ""
+
+        outlines = prefix.join("out").read().split("\n")
+        reader = 'loci.formats.in.FakeReader'
+        assert outlines[-2] == str(fakefile)
+        assert outlines[-3] == \
+            "# Group: %s SPW: false Reader: %s" % (str(fakefile), reader)
+
+    def testYamlOutput(self, tmpdir, capfd):
+
+        import yaml
+        from StringIO import StringIO
+
+        fakefile = tmpdir.join("test.fake")
+        fakefile.write('')
+        self.add_client_dir()
+        self.args += ["-f", "--output=yaml", str(fakefile)]
+        self.cli.invoke(self.args, strict=True)
+
+        o, e = capfd.readouterr()
+        result = yaml.load(StringIO(o))
+        result = result[0]
+        assert "fake" in result["group"]
+        assert 1 == len(result["files"])
+        assert "reader" in result
+        assert "spw" in result
+
+    def testBulkNoPaths(self):
+        t = path(__file__) / "bulk_import" / "test_simple"
+        b = t / "bulk.yml"
+        self.add_client_dir()
+        self.args += ["-f", "---bulk=%s" % b, "dne.fake"]
```

Comment:
Don't think this was specifically for compatibility testing. More likely just missed.